sources/packages/web/src/components/common/Reports.vue

```diff
@@ -56,6 +60,26 @@ export default defineComponent({
         props.reportList,
       );
     };
+    const formChanged = async (form: FormIOForm, event: FormIOChangeEvent) => {
+      // Populates the program year select component if required.
+      if (event.changed?.component.key === REPORT_TYPE_DROPDOWN_KEY) {
```

Comment:
If `event.changed` will be considered nullable the interface `FormIOChangeEvent` must be changed to represent it.
Did you find any scenario that would indicate that it can be null?

Reply:
Selecting the same value in the dropdown didn't trigger the `formChanged` event handler. I don't think that `event.changed` will be null ever when the `formChanged` event handler is called. Hence, removing the `?`.